prac_05/emails.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404 - Prac 5
+Program: Emails
+"""
+
+def main():
+    """Create dictionary of emails to names, and displays it."""
+    email_to_name = {}
+    email = input('Email: ')
+
+    while email != "":
+        name = extract_name(email)
+        name_confirmation = input(f"Is your name {name}? (Y/n) ").upper()
+        if name_confirmation != "" and name_confirmation != "Y":
```

Comment:
You could simplify this condition by directly checking for name_confirmation == "N" instead.